### DISCUSSION TITLE
Dependency question

### DISCUSSION THREAD

**Discussion by ghost** [fzyzcjy/flutter_convenient_test#385] [General]
I am attempting to write my first test and the example code uses `convenientTestMain` and `ConvenientTestSlot` which are only available via `convenient_test_dev` package so I got stuck a bit, as I missed [this being mentioned in the readme](https://github.com/fzyzcjy/flutter_convenient_test?tab=readme-ov-file#getting-started) which instructs to add via `dev_dependencies` of pubspec.yaml. My bad for not reading the getting started section 😆 

I have fixed this, but am now left wondering why `convenient_test` is required in `dependencies` at all?

**Comment by fzyzcjy** [comment]
Just play with it, and if it complains (probably just compilation error), add the dependency, otherwise no worries :)

**Reply by ghost** [reply]
Ah I found that we need it to get the `ConvenientTestWrapperWidget` to wrap the app root and get tests to work otherwise I get the error
```
flutter: The following _Exception was thrown while running async test code:
flutter: Exception: Please put `ConvenientTestWrapperWidget` in the widget tree near root, or ensure
flutter: ConvenientTestWrapperWidget.convenientTestActive=true
``` 

Another thing that would be good to have the in the documentation :)
